src/assets/js/archive.js

```diff
@@ -244,16 +278,28 @@ function getProvenanceLink(os, release, date, channel) {
 
 
 // Send requests to render the tables.
-$(function () {
-  if ($('#sdk-archives').length) {
+(() => {
```

Comment:
Nice idea to maintain the previous behavior with [IIFEs](https://developer.mozilla.org/en-US/docs/Glossary/IIFE).